factcast-site/documentation/content/usage/java/transformation.md

```diff
@@ -0,0 +1,81 @@
++++
+draft = false
+title = "Transformation"
+description = ""
+weight = 210
+
+creatordisplayname = "Uwe Schaefer"
+creatoremail = "uwe@codesmell.de"
+
+[menu.main]
+parent = "usage"
+identifier = "transformation"
+
++++
+
+
+## stay compatible
+
+When decoupling services via Events, it is vitally important, that the consuming party understands the Events it is interested in. Therefore, evolution is a challenge. As soon, as the publisher starts publishing a particular fact type in a (non-compatible) format, the consumer will break. This leads to complex deployment dependencies, that we tried to avoid in the first place.
```

Comment:
is capital letter here right for "event"? Or better write "fact" anyway? Same holds for fact. It is written sometimes with capital letter sometimes without